scapy/layers/usb.py

```diff
@@ -0,0 +1,231 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Philippe Biondi <phil@secdev.org>
+# Copyright (C) Gabriel Potter <gabriel@potter.fr>
+# This program is published under a GPLv2 license
+
+"""
+Default USB frames & Basic implementation
+"""
+
+# TODO: support USB headers for Linux and Darwin (usbmon/netmon)
+# https://github.com/wireshark/wireshark/blob/master/epan/dissectors/packet-usb.c  # noqa: E501
```

Comment:
Is this link needed here ? The same one is also included below.

Reply:
They have a different meaning: the first one shows what could be implemented on other platforms, and how it was done on wireshark.

The second one is the source of a dictionary I retrieved from wireshark